templates/flavors/base/cluster-template.yaml

```diff
@@ -21,12 +21,12 @@ kind: AzureCluster
 metadata:
   name: ${CLUSTER_NAME}
```

Comment:
Could we default cluster name?

Reply:
don't know about that one :)

Reply:
I'm split on that one... On one end it does allow a way easier "first cluster" experience. BUT each cluster name should be unique (so the Azure resources that use cluster name are also named uniquely) so your second cluster would fail, and it also requires syncing that with the cluster name you give to `clusterctl config cluster <cluster-name` in the quickstart.

Reply:
I think its useful for the user to set the CLUSTER_NAME so when/if they are creating multiple clusters they know where the name came from, also helps them find their resources on Azure. It would feel like too much magic to me

Reply:
How about we start soft, and increment gradually to more magic later?